CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/io/DataAccessParametersIoProvider.java

```diff
@@ -90,7 +91,53 @@ public static void saveParameters(final TabPane tabs) {
      * @param dataAccessPane the pane to load the JSON parameter file into
      * @see JsonIO#loadJsonPreferences(Optional, TypeReference) 
      */
+    
+    public static void loadParameters(final DataAccessPane dataAccessPane, String keyboardShortcut) {
+            final List<DataAccessUserPreferences> loadedParameters = JsonIO
+                .loadJsonPreferencesWithFilePrefix(
+                        Optional.of(DATA_ACCESS_DIR), Optional.of(keyboardShortcut),
+                        new TypeReference<List<DataAccessUserPreferences>>() {}
+                );
+            
+            if (loadedParameters != null) {
+            dataAccessPane.getDataAccessTabPane().removeTabs();
```

Comment:
could you indent the code in this if block?

Reply:
done